.github/workflows/deploy.yaml

```diff
@@ -285,6 +294,15 @@ jobs:
           HOPR_IMAGE_VERSION: ${{ steps.set-image-version.outputs.vsn }}
           HOPR_PACKAGE_VERSION: ${{ steps.set-package-version.outputs.vsn }}
 
+      - name: Verify the correct hoprd package version has been bundled
+        run: |
+          declare v=$(docker run --pull always ${HOPR_DOCKER_IMAGE}:${HOPR_IMAGE_VERSION} --version 2> /dev/null | sed -n '3p')
```

Comment:
Isn't `2>&1` more traditional ?

Reply:
Wouldn't be the same behaviour though. In this case I just want to get rid of stderr.